client/my-sites/stats/wordads/index.jsx

```diff
@@ -0,0 +1,192 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import page from 'page';
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { localize, numberFormat } from 'i18n-calypso';
+import moment from 'moment';
+
+/**
+ * Internal dependencies
+ */
+import DocumentHead from 'components/data/document-head';
+import Main from 'components/main';
+import StatsNavigation from 'blocks/stats-navigation';
+import StatsPeriodNavigation from '../stats-period-navigation';
+import DatePicker from '../stats-date-picker';
+import SidebarNavigation from 'my-sites/sidebar-navigation';
+import WordAdsChartTabs from '../wordads-chart-tabs';
+import titlecase from 'to-title-case';
+import PageViewTracker from 'lib/analytics/page-view-tracker';
+import JetpackColophon from 'components/jetpack-colophon';
+import WordAdsEarnings from './earnings';
+import { getSelectedSite, getSelectedSiteId, getSelectedSiteSlug } from 'state/ui/selectors';
+import { recordGoogleEvent } from 'state/analytics/actions';
+import PrivacyPolicyBanner from 'blocks/privacy-policy-banner';
+import StickyPanel from 'components/sticky-panel';
+
+class WordAds extends Component {
+	constructor( props ) {
+		super( props );
+		this.state = {
```

Comment:
this can be a class property assignment, which would let us drop the constructor